scribble-doc/scriblib/scribblings/figure.scrbl

```diff
@@ -85,14 +85,28 @@ The @racket[left] binding is a synonym for @racket[left-figure-style],
 provided for backward compatibility.}
 
 
-@defproc[(figure-ref [tag string?] ...+) element?]{
+@defproc[(figure-ref [tag string?] ...+
+                     [#:link-render-style link-style (or/c link-render-style? #f)])
+         element?]{
+
+Generates a reference to one or more figures, using a lowercase word ``figure''.
+
+If @racket[link-style] or (at the time of rendering)
```

Comment:
Is this "at the time of rendering" caveat the reason `secref` etc. don't use `(current-link-render-style)` as a default argument?

Reply:
Yes